### ISSUE TITLE
An incorrect error when extending a placeholder class containing @media rules

### ISSUE THREAD

**Issue by mzgol** [sass/libsass#1023]
The following SCSS:
```scss
%a {
    display: flex;
}

%b {
    @extend %a;

    @media screen {
        div {
            display: flex;
        }
    }
}

a {
    @extend %b;
}
```
causes an error:
```
>> You may not @extend an outer selector from within @media.
>> You may only @extend selectors within the same directive.
>> From "@extend %b" on line 16 of app/core-styles/_accordion.scss
>> 
>>   Line 9  Column 12  app/core-styles/_accordion.scss
```
As you can see, nothing is extended from within a media query. Ruby Sass parses it fine.

node-sass `3.0.0-beta.4`.

**Comment by mgreter** [comment]
Already fixed in latest master and 3.2.0-beta.4 (node-sass 3.0.0-beta.4 uses libsass 3.2.0 beta.2) !

**Comment by mzgol** [comment]
Great, thanks! :)
